### ISSUE TITLE
browser-resolve 1.7.0 breaks pkg.__dirname

### ISSUE THREAD

**Issue by ef4** [substack/module-deps#67]
If you try to use module-deps with browser-resolve 1.7.0, the value of `pkg.__dirname` emitted with the `'package'` event can be wrong. Instead of giving the actual directory of the package, it's giving the filename inside the package.

**Comment by mattfysh** [comment]
noticed this too - for now, I'm using the mothership package to traverse back to the correct __dirname

**Comment by mattfysh** [comment]
dagnamit. now I can't use relative transform paths, as module-deps tries to use the incorrect pkg.__dirname to resolve them.

**Comment by mattfysh** [comment]
Converted all relative transforms to node modules, and now everything appears to be working again. To fix the other issue using mothership:
```
var pkgPath = mothership.sync(pkg.__dirname, Boolean).path;
```

**Comment by zertosh** [comment]
Fixed in https://github.com/substack/module-deps/pull/81 and published as 3.7.9
